luxtronik/discover.py

```diff
@@ -2,6 +2,8 @@
 import logging
 import socket
 
+LUXTRONIK_DISCOVERY_PORTS = (4444, 47808)
```

Comment:
This should be a list from my understanding:

```suggestion
LUXTRONIK_DISCOVERY_PORTS = [4444, 47808]
```